karspexet/ticket/payment.py

```diff
@@ -104,3 +96,30 @@ def _send_mail_to_customer(self):
             [to_address],
             fail_silently=False,
         )
+
+class FakePaymentProcess(PaymentProcess):
+
+    def _charge_card(self):
+        if self.data['payment_success'] == 'true':
+            return True
+        else:
+            raise PaymentError("NO TICKETS FOR YOU")
+
+
+class StripePaymentProcess(PaymentProcess):
+
+    def _charge_card(self):
+        amount = self.reservation.total_price() * 100 # Öre
+        stripe_token = self.data['stripeToken']
+
+        try:
+            return stripe.Charge.create(
+                source=stripe_token,
+                amount=amount,
+                currency="sek",
+                description="Biljetter till Kårspexet"
+            )
+        except (APIConnectionError, AuthenticationError, InvalidRequestError, RateLimitError, StripeError) as e:
+            # Too many requests made to the API too quickly
+            logger.error(e)
+            raise PaymentError(str(e))
```

Comment:
Instead of this, which loses the stack trace, use `raise PaymentError from error`, https://docs.python.org/3/reference/simple_stmts.html#raise

That or the `with_traceback` method, whichever makes most sense here.